openshift2nulecule/cli/main.py

```diff
@@ -46,19 +46,52 @@ def __init__(self):
                                  help="Show debug messages",
                                  action='store_true')
 
+        self.parser.add_argument("--oc-registry-host",
+                                 help="Hostname of exposed internal OpenShift registry",
+                                 required=False)
+
+        self.parser.add_argument("--export-images",
+                                 help="Pull images from OpenShift registry to "
+                                      "local Docker instance and push them to "
+                                      "remote registry (--registry-host) ",
+                                 choices=["none", "internal", "all"],
+                                 default="none",
+                                 required=False)
+
+        self.parser.add_argument("--registry-host",
+                                 help="External registry hostname",
```

Comment:
hostname or IP address, what if this is not specified? Does this assumes that the `localhost` is taken by default?